templates/imdb/templates.yaml

```diff
@@ -0,0 +1,56 @@
+dataset: imdb
+templates:
+  02ff2949-0f45-4d97-941e-6fa4c0afbc2d: !Template
+    id: 02ff2949-0f45-4d97-941e-6fa4c0afbc2d
+    jinja: The following movie review expresses what sentiment? {{text}} ||| {{ ["negative",
+      "positive"] [label] }}
+    name: imdb_5
+    reference: ''
+  866474a5-1498-46b7-bfee-ac0c5160707f: !Template
+    id: 866474a5-1498-46b7-bfee-ac0c5160707f
+    jinja: '{{text}}How does the viewer feel about the movie? ||| {{ ["negative",
+      "positive"] [label] }}'
+    name: imdb_6
+    reference: ''
+  96538f30-f2c1-430e-8fc6-936a16966d9c: !Template
+    id: 96538f30-f2c1-430e-8fc6-936a16966d9c
+    jinja: '{{text}} What sentiment does the writer express for the movie? ||| {{
+      ["negative", "positive"] [label] }}'
+    name: imdb_7
+    reference: ''
+  af51297c-38a3-4d6c-a8b5-04b1243d7443: !Template
+    id: af51297c-38a3-4d6c-a8b5-04b1243d7443
+    jinja: '{{text}}The sentiment expressed for the movie is ||| {{ ["negative", "positive"]
+      [label] }}'
+    name: imdb_3
+    reference: ''
+  b93b74ac-fe95-40b4-9610-318b46ab820f: !Template
+    id: b93b74ac-fe95-40b4-9610-318b46ab820f
+    jinja: '''{{text}} What is the sentiment expressed in this text? ||| {{ ["negative",
+      "positive"] [label] }}'''
```

Comment:
Not sure if it's some kind of Jinja escaping thing or something entered in the app, but these render with extra single quotes around the prompt and output.